src/game/GameResults.as

```diff
@@ -134,6 +134,7 @@ package game
         {
             // Add keyboard navigation
             stage.addEventListener(KeyboardEvent.KEY_DOWN, eventHandler);
+            stage.addEventListener(KeyboardEvent.KEY_UP, eventHandler);
```

Comment:
@Psycast Do I need to add a remove event listener to the `else if (keyCode == _gvars.playerUser.keyQuit)` section which has one for KEY_DOWN?

Reply:
Irrelevant based on comments in discord.